README.md

```diff
@@ -12,8 +12,8 @@ It's written in Golang and React and runs as a single Linux binary with MySQL or
 
 ## Try out Mattermost
 
-- [Join the Mattermost Contributor's server](https://pre-release.mattermost.com/) (latest nightly builds, unstable)
-- [Join the Mattermost Demo server](https://demo.mattermost.com/signup_email) (latest stable version)
+- [Join the Mattermost Contributor's server](https://community.mattermost.com/) (latest nightly builds, unstable. Get into contact with us and our community here.)
```

Comment:
```suggestion
- [Join the Mattermost Contributor's server](https://community.mattermost.com/signup_user_complete/?id=f1924a8db44ff3bb41c96424cdc20676) to join community discussions about contributions, development and more

Reply:
The community.mattermost.com is now also on the latest stable version, so no need to call it out specifically